pl_examples/intel_compressor/README.md

```diff
@@ -0,0 +1,82 @@
+# Step-by-Step
+
+This document describes the step-by-step instructions for reproducing PyTorch ResNet50/ResNet18/ResNet101 tuning results with [Intel® Neural Compressor](https://github.com/intel/neural-compressor).
+
+# Prerequisite
+
+### 1. Installation
+
+```shell
+pip install -r requirements.txt
+```
+
+### 2. Prepare Dataset
+
+Download [ImageNet](http://www.image-net.org/) Raw image to dir: /path/to/imagenet.  The dir include below folder:
+
+```bash
+ls /path/to/imagenet
+train  val
+```
+
+# Quantize the model
+
+### 1. ResNet50
+
+```shell
+python imagenet.py --arch resnet50 --pretrained --data-path /path/to/imagenet --batch-size 300 --default_root_dir ./checkpoint
+```
+
+> **Note**
+>
+> batch size should be the munber which is sampling size divisible by. sampling size is a setting in config/quantization.yaml
+
+### 2. ResNet18
+
+```shell
+python imagenet.py --pretrained --data-path /path/to/imagenet --batch-size 300 --default_root_dir ./checkpoint
+```
+
+### 3. ResNext101_32x8d
+
+```shell
+python imagenet.py --arch resnext101_32x8d --pretrained --data-path /path/to/imagenet --batch-size 300 --default_root_dir ./checkpoint
+```
+
+### 4. InceptionV3
+
+```shell
+python imagenet.py --arch inception_v3 --pretrained --data-path /path/to/imagenet --batch-size 300 --default_root_dir ./checkpoint
+```
+
+# Benchmark
+
+### int8 performance:
+
+python imagenet.py --arch resnet50 --pretrained --data-path /path/to/imagenet -e  --default_root_dir ./checkpoint
+
+# Saving and loading model:
+
+- Saving model:
+  After tuning with Neural Compressor, we can get neural_compressor.model:
+
+```
+quantizer = pl.IncQuantization(
+        'config/quantization.yaml',
+        metric_name="val_acc1",
```

Comment:
```suggestion
        monitor="val_acc1",
```

We use monitor in our components, let's keep it consistent.

Reply:
changed